scripts/tests/chiptest/test_definition.py

```diff
@@ -383,13 +386,27 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                     runner.RunSubprocess(python_cmd, name='CHIP_REPL_YAML_TESTER',
                                          dependencies=[apps_register], timeout_seconds=timeout_seconds)
             else:
-                pairing_cmd = paths.chip_tool_with_python_cmd + ['pairing', 'code', TEST_NODE_ID, setupCode]
+                pairing_server_args = []
+                if tool_hci_number is not None:
+                    pairing_cmd = paths.chip_tool_with_python_cmd + [
+                        "pairing",
+                        "code-wifi",
+                        TEST_NODE_ID,
+                        "Virtual_Wifi",
+                        "ExamplePassword",
+                        "MT:-24J042C00KA0648G00",
```

Comment:
Doesn't that need to be `setupCode` or some derivative of it instead of a hardcoded thing?  Or at least document why this is being hardcoded (and e.g. is therefore hardcoding a specific discriminator).